KlimaDAO/klimadao
# auto-compounder feature

## Description

[](https://github.com/KlimaDAO/klimadao/issues/2393)

## Related Ticket

Closes [2393](https://github.com/KlimaDAO/klimadao/issues/2393)


## How to Test
- Go to URL
- View Deposit list to see eligible tokens to stake to 
- Go to Your Positions tab to view All positions and then Attempt to withdraw. 

## Notes For QA
1. Test AutoCompounder Deposit + Withdraw flows
2. Test Existing Cross Chain Retirement 

Specific pages, components or journeys that might be affected:
- Base Retirement section of the existing base dapp

Relevant preview URLs:
- 

Other notes:
-
